### PR TITLE
Added  Example Inventory

### PR DESCRIPTION
**Pull request by patsevanton**

### CODE REVIEW

**Comment by patsevanton** [comment]
Please Review PR

File: README.md [hunk 1]
```diff
@@ -28,6 +28,28 @@ Coming soon.
...
+      patroni1
+      patroni2
+      patroni3
+
+      [etcd]
+      patroni1
+      patroni2
+      patroni3
+
+      [etcd-master]
```
Comment (kostiantyn-nemchenko) [inline README.md]: What's the reason to add this group here? The retr0h.etcd role suggested as dependency works without it.

Comment (patsevanton) [reply README.md]: playbook retr0h.etcd wont work

Comment (kostiantyn-nemchenko) [reply README.md]: Ok, I will take a look at it to get more details. Thank you

**Pull request closed without merging**